config/master/manifests.jsonnet

```diff
@@ -390,3 +409,4 @@ byRegion("cni-metrics-helper", metricsHelper);
   [kv[0] + ".yaml"]: std.manifestYamlStream(objectValues(kv[1]))
   for kv in objectItems(output)
 }
+{"aws-k8s-cni-configmap.yaml": std.manifestYamlStream(objectValues(awsnodecm))}
```

Comment:
(My previous attempt at this comment disappeared?)

Nit: Instead of this, append to `output` above.  Something like:
```jsonnet
local output = ... + {"aws-k8s-cni-configmap": awsnodecm};
```

Reply:
(Sorry for the delay), changed the jsonnet file as per your suggestion, thanks @anguslees :)